.changeset/silent-trainers-guess.md

```diff
@@ -0,0 +1,10 @@
+---
+'@primer/react-brand': patch
+---
+
+Updated Hero component stories for accessibility
+
+- Removed unnecessary `aria-labeledby` on the section component
+- Removed the related `id` on the heading component
+- Removed `br` from Hero feature story which is inaccessible
```

Comment:
This one can be removed, as it doesn't affect package output

```suggestion
```

Reply:
Updated in [SHA](https://github.com/primer/brand/pull/336/commits/14239ebd9cc7aa4f111f72d7a698c69f04c65778)